spec/lib/popular_article_spec.rb

```diff
@@ -0,0 +1,60 @@
+require File.join(Rails.root, 'lib', 'popular_article')
+
+describe PopularArticle do
+  let!(:blog) { create(:blog) }
+
+  subject { described_class }
+
+  describe '#find' do
```

Comment:
Ow, I see. But even with that, I would use the Ruby documentation convention of . (or ::) when referring to a class method's name and # when referring to an instance method's name. And I would move the subject line inside of the describe block too. Wdyt?